docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>The code fails if we don't specify a species_list. We should set it up such that it behaves more like the SDEC plot. i.e., we should make it plot all elements if no species_list is defined. And we should also add an elements flag to allow us to plot e.g., the 10 most interacting elements. This is all implemented in the SDEC plotter so should be easy to see how it is implemented there for ease of reproduction here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>